lib/terraforming/resource/efs.rb

```diff
@@ -0,0 +1,56 @@
+module Terraforming
+  module Resource
+    class EFSFileSystem
+      include Terraforming::Util
+
+      def self.tf(client: Aws::EFS::Client.new)
+        self.new(client).tf
+      end
+
+      def self.tfstate(client: Aws::EFS::Client.new)
+        self.new(client).tfstate
+      end
+
+      def initialize(client)
+        @client = client
+      end
+
+      def tf
+        apply_template(@client, "tf/elastic_filesystem")
+      end
+
+      def tfstate
+        file_systems.each_with_index.inject({}) do |resources, (efs, idx)|
```

Comment:
__[rubocop]__
* Unused block argument - `idx`. If it's necessary, use `_` or `_idx` as an argument name to indicate that it won't be used. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)

<!-- Comment by [SideCI](https://sideci.com) -->